src/main/java/mate/academy/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import mate.academy.db.Storage;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        if (manufacturer != null) {
```

Comment:
let's remove this check for null. 
If you pass null you will not notice any mistake, but method won't work correctly because nothing has been saved to db.